bigtable-client-core-parent/bigtable-hbase-integration-tests-common/src/test/java/com/google/cloud/bigtable/hbase/AbstractTestListTables.java

```diff
@@ -182,6 +187,67 @@ public void testNotFound() throws IOException {
     }
   }
 
+  @Test
+  public void testListTablesWithEmptyElement() throws IOException {
+    try (Admin admin = getConnection().getAdmin()) {
+      sharedTestEnv.createTable(sharedTestEnv.newTestTableName());
+      Exception actualError = null;
+      try {;
+        assertTrue(admin.listTables((Pattern) null).length > 0);
+      } catch (Exception e) {
+        actualError = e;
+      }
+      assertNull(actualError);
+
+      try {
+        admin.listTables((String) null);
+      } catch (Exception e) {
+        actualError = e;
+      }
+      assertNotNull(actualError);
+      assertTrue(actualError instanceof NullPointerException);
+      actualError = null;
+
+      try {
+        assertEquals(0, admin.listTables("").length);
+      } catch (Exception e) {
+        actualError = e;
+      }
+      assertNull(actualError);
+    }
+  }
+
+  @Test
+  public void testListTableNamesWithEmptyElement() throws IOException {
+    try (Admin admin = getConnection().getAdmin()) {
+      sharedTestEnv.createTable(sharedTestEnv.newTestTableName());
+      Exception actualError = null;
+      try {
```

Comment:
I don't think the try is necessary here. Also please move actualError after the assertTrue